lib/cartopy/tests/test_crs.py

```diff
@@ -186,6 +186,16 @@ def test_globe(self):
         assert_arr_almost_eq(rugby_pt, (1400915, 1741319), decimal=0)
         assert_arr_almost_eq(footy_pt, (155657, 193479), decimal=0)
 
+    def test_globe_defaults(self):
+        """Regression test for #564"""
+        default_globe = ccrs.Globe()
+        assert default_globe.ellipse == 'WGS84'
+
+        eccentric_globe = ccrs.Globe(semimajor_axis=1000, semiminor_axis=500)
+        assert not eccentric_globe.ellipse  # would need revision with #330
```

Comment:
What does this mean? What kind of revision?

Reply:
If I understood #330 correctly, when #330 gets implemented, this unit test might start misbehaving, because the `.ellipse` attribute might get set with some kind of default or whatever gets passed along from `proj4`. The `assert` statement would get invalidated in that case and would have to be revised, right?

Reply:
I see! I'll simply remove the comment then.